### ISSUE TITLE
Apparently, this is broken again.

### ISSUE THREAD

**Issue by markfilipak-windows** [mpv-player/mpv#6891]
Apparently, this is broken again.
A snapshot of DVD (720x480) is 720x540 from both the actual DVD & an MKV made from it.
Expected: 720x480.

C:\CMD & tiny apps\MPV>mpv -V
mpv 0.29.1 Copyright © 2000-2018 mpv/MPlayer/mplayer2 projects
 built on UNKNOWN
ffmpeg library versions:
   libavutil       56.19.101
   libavcodec      58.30.100
   libavformat     58.18.100
   libswscale      5.2.100
   libavfilter     7.31.100
   libswresample   3.2.100
ffmpeg version: 4.1+1209.ge320f9576a

_Originally posted by @markfilipak-windows in https://github.com/mpv-player/mpv/issues/1007#issuecomment-524668256_

**Comment by markfilipak-windows** [comment]
I added to 'https://github.com/mpv-player/mpv/issues/1007' and then noticed that it was closed.
I sought a way to reopen the issue, but could only find "Reference in new issue".
I clicked it and "Reference in new issue" dialog opened.
I clicked the "Create Issue" button.

Akemi closed that issue with the complaint that I didn't "make a proper issue".

I don't give a damn about learning how to run github.
